releasenotes/notes/placeholder.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - 50655
+releaseNotes:
+- |
+  **Added** Add --verbosity flag to istioctl x ztunnel-config to allow control over the verbosity of the output.
```

Comment:
```suggestion
  **Added** Add --verbosity flag to istioctl ztunnel-config to allow control over the verbosity of the output.
```

Reply:
Didn't mean to commit this (it's from another local branch I was working on, will remove)